src/main/java/org/team1540/bobafett/commands/drivetrain/TankDrive.java

```diff
@@ -24,7 +27,8 @@ public void execute() {
         double right = Math.abs(controller.getRightY()) > deadzone ? Math.pow(controller.getRightY(), 3) : 0;
         double forward = Math.abs(controller.getRightTriggerAxis()) > deadzone ? Math.pow(controller.getRightTriggerAxis(), 3) : 0;
         double backward = Math.abs(controller.getLeftTriggerAxis()) > deadzone ? Math.pow(controller.getLeftTriggerAxis(), 3) : 0;
-        drivetrain.setPercent(0.5*(left + forward + backward), 0.5*(right + forward + backward));
+        double multiplier = elevator.getRotations() < 60 ? 0.5 : 0.25;
```

Comment:
Why a stepwise function here? Could be nice to do linearly

Reply:
I didn't actually think of that, but I should probably do that